plugin/src/main/java/io/jenkins/plugins/analysis/core/model/StaticAnalysisLabelProvider.java

```diff
@@ -208,6 +209,13 @@ public String toString() {
      * @return the name of the side panel link
      */
     public String getLinkName() {
+        if (Boolean.TRUE.equals(UserExperimentalFlag.getFlagValueForCurrentUser("jenkins.model.experimentalflags.NewBuildPageUserExperimentalFlag"))) {
```

Comment:
I'm not sure if this call requires to have a running Jenkins instance but since the tests are not failing the call to Jenkins is not always done.